core/pom.xml

```diff
@@ -12,6 +12,7 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
+            <scope>provided</scope>
```

Comment:
wait who will provide this now? if core is about to export this, it needs to have it on the runtime classpath?

Reply:
nice catch. this was unintentional.